src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/NoOpRestoreResult.cs

```diff
@@ -30,21 +30,21 @@ class NoOpRestoreResult : RestoreResult
 
             if (isTool)
             {
-                log.LogDebug(string.Format(CultureInfo.CurrentCulture,
-                        Strings.Log_ToolSkippingLockFile,
+                log.LogInformation(string.Format(CultureInfo.CurrentCulture,
+                        Strings.Log_ToolSkippingAssetsFile,
                         LockFilePath));
-                log.LogDebug(string.Format(CultureInfo.CurrentCulture,
+                log.LogInformation(string.Format(CultureInfo.CurrentCulture,
```

Comment:
nit: I think the skip for tool and project could be Verbose. I don't think it needs to be under info, nuget.exe still prints at the info level and this might be a bit much to show all these.